tests/e2e/core/pxc/65-assert.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kuttl.dev/v1
+kind: TestAssert
+timeout: 600
+---
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseCluster
+metadata:
+  name: test-pxc-cluster
+status:
+  status: ready
```

Comment:
Why isn't this part of `64-assert.yaml`?

Reply:
We want to first check that it gets to `resizingVolumes` state, which we do in `64-assert.yaml`. Next, we want to check that resizing was complete and DB is in `ready` state, which is done in `65-assert.yaml`.

Can a single assert step sequentially check it the DB is first in `resizingVolumes` and then `ready`? I was under the impression that for an assertion to pass, ALL the conditions have to be met (which would fail here because the DB cannot be in both `resizingVolumes` and `ready` at once) 🤔

Reply:
Ahhh nevermind, I forgot about the `resizingVolumes` state, I assumed that the only thing you were checking in `64-assert.yaml` was the new size.